identity/config.go

```diff
@@ -0,0 +1,20 @@
+package identity
+
+import "perun.network/go-perun/wallet"
+
+// WalletConfig defines wallet specific configuration parameters.
+type WalletConfig struct {
+	Addr         wallet.Address
+	KeystorePath string
+	Pwd          string //The passwords are received by the cli app as command line arguments are passed to the config.
+}
+
+// UserConfig defines user specific configuration parameters.
+type UserConfig struct {
+	Alias string
+
+	ListenerURL URL //Client will lister for incoming network connections at this URL
```

Comment:
This is the endpoint for the perun-node, not the API, right? Otherwise one could think, it is the same endpoint for both.

Reply:
Yes, it is the endpoint for perun-node. I have renamed the variable to `NetworkAddress` to make it more clear.